src/types/cart.ts

```diff
@@ -0,0 +1,2 @@
+
+export const ADD_TO_CART = 'ADD_TO_CART';
```

Comment:
Oh last thing - one typescripty way this can be done is to use a string enum for this, helps to provide some organization as it grows.

```
export enum CartTypes {
  AddToCart = 'ADD_TO_CART',
}
```